qiskit_ibm_runtime/qiskit_runtime_service.py

```diff
@@ -986,6 +986,13 @@ def run(
             RuntimeProgramNotFound: If the program cannot be found.
             IBMRuntimeError: An error occurred running the program.
         """
+        # TODO: Remove this after 3 months
+        if program_id in ["hello-world", "vqe", "qaoa"]:
+            raise IBMInputValueError(
+                "The hello-world, vqe, and qaoa programs have been retired in the "
+                "Qiskit Runtime service. "
+                "Please visit https://qiskit.org/ecosystem/ibm-runtime for more information."
```

Comment:
I think we can give a bit more information in the text. Something like 
```
Please visit https://qiskit.org/ecosystem/ibm-runtime for an introduction on Sessions and Primitives, and to access tutorials on how to execute VQE and QAOA using Qiskit Runtime Primitives.
```

Reply:
Sure, I'll leave out `hello-world` for now and add it back next sprint when it's actually removed